tests/system/test_database_api.py

```diff
@@ -289,6 +289,47 @@ def test_update_ddl_w_default_leader_success(
     assert len(temp_db.ddl_statements) == len(ddl_statements)
 
 
+def test_create_role_grant_access_success(
```

Comment:
Integration test for list_database_roles?

Reply:
not needed I think, it's part of admin api, already get's tested on the way that it gets tested. Should we add one ?

Reply:
Added one.